package.json

```diff
@@ -42,7 +42,9 @@
     "publish-release-next": "ng run tools-release:build-publish-release-next && node dist/libs/tools/publish-release-next/main.js",
     "barista:prerender": "ng run barista-design-system:prerender --routesFile dist/barista-data/routes.txt",
     "bazel:format": "bazel build //:buildifier",
-    "bazel:build": "bazel build //libs/workspace/src/builders/barista-build --remote_upload_local_results=true --google_credentials=tools/bazel_build_cache/cache-write-key.json"
+    "bazel:build": "bazel build //libs/workspace/src/builders/barista-build --remote_upload_local_results=true --google_credentials=tools/bazel_build_cache/cache-write-key.json",
+    "api-guard:generate": "ts-api-guardian --outDir api_guard --rootDir dist/libs/components/ --allowModuleIdentifiers Highcharts dist/libs/components/*/index.d.ts",
+    "api-guard:verify": "node dist/libs/workspace/src/scripts/api-guard-verify.js --verifyDir api_guard --rootDir dist/libs/components/ --allowModuleIdentifiers Highcharts dist/libs/components/*/index.d.ts"
```

Comment:
why do we have to provide Highchcarts as moduleIdentifier and only Highcharts?

Reply:
Highcharts is the only module within the components, that we need to import as `import * as Highcharts fomr 'highcharts';` These imports need to be in the allowList for the API guardian, i guess.